app/src/bin/crate.in.sh

```diff
@@ -80,4 +80,7 @@ JAVA_OPTS="$JAVA_OPTS -XX:+DisableExplicitGC"
 JAVA_OPTS="$JAVA_OPTS -Dfile.encoding=UTF-8"
 
 # Use our provided JNA always versus the system one
-JAVA_OPTS="$JAVA_OPTS -Djna.nosys=true"
\ No newline at end of file
+JAVA_OPTS="$JAVA_OPTS -Djna.nosys=true"
```

Comment:
We've a SQL API to change the loglevel at runtime. So I think it would be okay to disable jmx.